plugins/inventory/lxd.py

```diff
@@ -41,6 +41,15 @@
             aliases: [ cert_file ]
             default: $HOME/.config/lxc/client.crt
             type: path
+        server_cert:
+            description:
+            - The server certificate file path.
+            - It is common for lxd servers to use self-signed certificates. Providing the server
+              certificate allows to do verification of the HTTPS connection. When this option is
+              used, the hostname part of the url doesn't need to match one of the Subject Alternate
+              Names (SANs) of the certificate. This is essentially certificate pinning.
```

Comment:
I don't see how this is essentially certificate pinning? You do not allow to provide hashes of the leaf certificate.

Reply:
Fair, trust anchor might have been a better way to express it. I dropped the sentence since it was not clear.